readme.md

```diff
@@ -82,6 +84,7 @@ _NB: Beyond session 1, these sessions aren't in any particular order. There's so
     - [The War on Drugs is an Epic Fail](http://www.nytimes.com/video/opinion/100000004642370/jay-z-the-war-on-drugs-is-an-epic-fail.html). Jay Z, Molly Crabapple, Jim Batt, Kim Boekbinder and dream hampton.
     - [This is what it will be like to protest in 2020, when the state is watching your every move](http://fusion.net/story/362575/protest-2020-future-surveillance-activism-police/). Peter Moskowitz.
     - [The Making of the American Police State](https://www.jacobinmag.com/2015/07/incarceration-capitalism-black-lives-matter/). Christian Parenti.
+    - [‘I was terribly wrong’ - writers look back at the Arab spring five years on](https://www.theguardian.com/books/2016/jan/23/arab-spring-five-years-on-writers-look-back). Various authors.
```

Comment:
I found this really interesting because it's an anthropological take on how a specific heralded technoutopia failed to materialize.